Navigation-Examples/Examples/Building-Extrusion.swift

```diff
@@ -0,0 +1,195 @@
+import UIKit
+import MapboxCoreNavigation
+import MapboxNavigation
+import MapboxDirections
+import Mapbox
+
+class BuildingExtrusionViewController: UIViewController, NavigationMapViewDelegate, NavigationViewControllerDelegate, MGLMapViewDelegate {
+    
+    var mapView: NavigationMapView?
+    
+    var routeOptions: NavigationRouteOptions?
+    
+    var currentRoute: Route? {
+        get {
+            return routes?.first
+        }
+        set {
+            guard let selected = newValue else { routes?.remove(at: 0); return }
```

Comment:
This creates unexpected behavior:
```
self.currentRoute = nil
if self.current.route == nil { // will be true if there were at least 2 `routes`
// ...
```
I suggest
```suggestion
            guard let selected = newValue else { routes = nil; return }
```

Reply:
Latest update should resolve this issue too.